packages/tools/package.json

```diff
@@ -30,7 +30,6 @@
   },
   "dependencies": {
     "@cornerstonejs/core": "^1.81.3",
-    "@icr/polyseg-wasm": "0.4.0",
```

Comment:
Can we avoid implementing this change in version 1.x, as it has been in use for a couple of months? Instead, let's consider using the peerImport only for WSI, which is relatively new. This approach would minimize the risk of introducing breaking changes for existing users.

In version 2.x we will peerImport polyseg too